src/zeit/nightwatch/pytest.py

```diff
@@ -47,5 +54,5 @@ def pytest_collection_modifyitems(items):
     `-m 'not selenium'`. (The fixture must be provided by the client project.)
     """
     for item in items:
-        if 'selenium' in item.fixturenames:
+        if 'selenium' in getattr(item, 'fixturenames', []):
```

Comment:
flake8 und Freunde lassen grüßen, vermutlich? Danke jedenfalls. :)

Reply:
Ich glaub es war nicht mal 'flake8' in dem Fall. Da kommt als `Item` einfach alles mögliche vorbei, Funktionen, Fixtures, Marker usw (und nicht alle haben `.fixturenames`)…